mission/functions/systems/consumables/fn_health_effects.sqf

```diff
@@ -26,26 +89,122 @@ else
 	player allowSprint true;
 };
 
-// force walk if player is hungry
+///////////////////////////////////////////////////////////////////////////////////////////
+// HUNGRY /////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////////////////////////////////////////////////////////////////
+
+// start player blacking out -- simulate tiredness
+if ((player getVariable ["vn_mf_db_hunger", 1]) <= 0.25) then {
+	private _blurValue = 0.25 - (player getVariable ["vn_mf_db_hunger", 1]);
+	private _blur = ["HungerBlur", ["dynamicBlur", 398]] call _fnc_get_or_create_vfx_handler;
+  _blur ppEffectEnable true;
+  _blur ppEffectAdjust [(random _blurValue) / 2];
+  _blur ppEffectCommit 1 + random 1;
+
+} else {
+	[["HungerBlur"]] call _fnc_delete_vfx_handlers;
+};
+
+// force walk if player is out of hunger stat
 if (player getVariable ["vn_mf_db_hunger", 1] isEqualTo 0) then
 {
 	_stamina_scheme = "Exhausted";
 	player forceWalk true;
+	if (random 1 < 0.2) then {["BLACK"] call _fnc_fade_inout_to_color};
 }
 else
 {
 	player forceWalk false;
 };
```

Comment:
as above, move the hungry block to a `consumables/fn_hunger_activate.sqf` file or some such?